extensions/arc/runtime/src/main/java/io/quarkus/arc/runtime/InterceptorBindings.java

```diff
@@ -13,4 +14,10 @@ public class InterceptorBindings {
     public static Set<Annotation> getInterceptorBindings(InvocationContext invocationContext) {
         return (Set<Annotation>) invocationContext.getContextData().get(ArcInvocationContext.KEY_INTERCEPTOR_BINDINGS);
     }
+
+    @SuppressWarnings("unchecked")
+    public static Set<AbstractAnnotationLiteral> getInterceptorBindingLiterals(InvocationContext invocationContext) {
```

Comment:
Could you pls add a simple comment why this method is needed?

Reply:
Done